### DISCUSSION TITLE
Organisation Webhooks

### DISCUSSION THREAD

**Discussion by M3rcena** [community/community#123833] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Hello everyone,

I added a webhook to my organisation so I can log every repository. But is it possible to exclude a repository from the webhook?

**Comment by Ethan-Software-Developer** [comment]
Yes it is possible

Go to the settings of the webhook in your GitHub organization.

Look for the section where you can specify the repositories the webhook applies to. This section may be labeled "Repository settings" or "Repository scope."

If the webhook is currently set to trigger for all repositories in the organization, you'll need to switch to the option to select specific repositories.

Select the repositories you want the webhook to apply to, excluding the repository you want to exclude from the list.

Save your changes.

**Reply by Ethan-Software-Developer** [reply]
I hope it helps you out

**Reply by M3rcena** [reply]
> Yes it is possible
> 
> Go to the settings of the webhook in your GitHub organization.
> 
> Look for the section where you can specify the repositories the webhook applies to. This section may be labeled "Repository settings" or "Repository scope."
> 
> If the webhook is currently set to trigger for all repositories in the organization, you'll need to switch to the option to select specific repositories.
> 
> Select the repositories you want the webhook to apply to, excluding the repository you want to exclude from the list.
> 
> Save your changes.

Hello!

There is no such category for repository settings inside the webhook

**Reply by rrundle-dave** [reply]
@Ethan-Software-Developer I don't see what you're talking about either. There are no Repository settings in the Webhook settings. This is all that is available. 
![image](https://github.com/community/community/assets/91917325/0ac67fc9-8bf0-4ee3-aa01-b5fe39948daa)

Can you provide a screenshot or a little more details?

**Reply by M3rcena** [reply]
yeah, thats what I said. I am still trying to find how to exclude a repository from the webhook

**Comment by gm3dmo** [comment, marked as answer]
There is no way to exclude a repository from an organization webhook

https://docs.github.com/en/webhooks/using-webhooks/creating-webhooks#creating-an-organization-webhook

To achieve this you'll need to exclude/discard the hook on the webhook endpoint.
